src/main/java/duke/command/FindCommand.java

```diff
@@ -0,0 +1,34 @@
+package duke.command;
+
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+import duke.task.Task;
+
+import java.util.ArrayList;
+
+public class FindCommand extends Command {
+    String keyword;
```

Comment:
Despite not really violating the coding standards as it only mentioned that variable shouldn't be public, but since other classes or sub-classes are not accessing the `keyword` variable, would it be better to declare it as private?